base_miner/UCF/loss/__init__.py

```diff
@@ -0,0 +1,13 @@
+import os
+import sys
+current_file_path = os.path.abspath(__file__)
+parent_dir = os.path.dirname(os.path.dirname(current_file_path))
+project_root_dir = os.path.dirname(parent_dir)
+sys.path.append(parent_dir)
+sys.path.append(project_root_dir)
```

Comment:
[Question] Same question with other __init__ files, I see these come straight from DeepfakeBench, but this is hacky af lol